src/gen_ecmult_gen_static_prec_table.c

```diff
@@ -10,8 +10,8 @@
 #include "../include/secp256k1.h"
 #include "assumptions.h"
 #include "util.h"
-#include "group.h"
-#include "ecmult_gen.h"
```

Comment:
nit: I think `ecmult_gen.h` is included for `ECMULT_GEN_PREC_G` and `ECMULT_GEN_PREC_N`.

Reply:
Fixed.